From 9c3ea662a066387e0460e099536d78ecfee60377 Mon Sep 17 00:00:00 2001
From: Alexey Brodkin <abrodkin@synopsys.com>
Date: Mon, 24 Mar 2014 17:15:50 +0400
Subject: [PATCH] axs101 - increase EEPROM page write delay from 32 to 64 msec

With 32 milliseconds delay on some boards EEMPROM got written inconsistently.
With 64 msec all of our existig boards show properly written EEPROM.

Cc: Tom Rini <trini@ti.com>

Signed-off-by: Alexey Brodkin <abrodkin@synopsys.com>
---
 include/configs/axs101.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/configs/axs101.h b/include/configs/axs101.h
index f6b569a..8d03110 100644
--- a/include/configs/axs101.h
+++ b/include/configs/axs101.h
@@ -103,7 +103,7 @@
 #define CONFIG_SYS_I2C_EEPROM_ADDR_LEN		1
 #define CONFIG_SYS_I2C_EEPROM_ADDR_OVERFLOW	1
 #define CONFIG_SYS_EEPROM_PAGE_WRITE_BITS	3
-#define CONFIG_SYS_EEPROM_PAGE_WRITE_DELAY_MS	32
+#define CONFIG_SYS_EEPROM_PAGE_WRITE_DELAY_MS	64
 
 /*
  * SD/MMC configuration
-- 
1.8.5.3

